app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ReviewsController < ApplicationController
+  def new
+    @booking = Booking.find(params[:booking_id])
+    @review = Review.new
+  end
+
+  def create
+    @review = Review.new(review_params)
+    @review.booking = Booking.find(params[:booking_id])
+    if @review.save
+      redirect_to booking_path(@review.booking)
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def review_params
+    params.require(:review).permit(:content, :rating, :booking_id)
```

Comment:
we can also remove booking_id on this but need to create a few more lines above in the create (similar to the bookings controller) let's have a look through together and test!